CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The Singularity Project has been
 and re-branded as Apptainer.
 For older changes see the [archived Singularity change log](https://github.com/apptainer/singularity/blob/release-3.8/CHANGELOG.md).
 
+## Changes Since Last Release
+
+### New features / functionalities
+
+- Support for a custom hashbang in the `%test` section of a Apptainer recipe
```

Comment:
I want to include this also in rc.3, so remove the 'New features / functionalities' label.   You could wait until the other PR is merged because the second one merged will need to be rebased anyway.

Also change `a Apptainer` to `an Apptainer`.